docs/advanced-analytics/pgvector-search.md

```diff
@@ -0,0 +1,429 @@
+---
+title: Use pgvector for Vector Similarity Search
+---
+
+# Use pgvector for Vector Similarity Search
+
+pgvector is an open-source plugin for vector similarity search. It supports both exact and approximate nearest neighbor searches, as well as L2 distance, inner product, and cosine distance. For more details, see [pgvector/pgvector: Open-source vector similarity search for Postgres](https://github.com/pgvector/pgvector). Apache Cloudberry allows you to use pgvector for data storage, querying, indexing, hybrid searches, and more through SQL statements.
```

Comment:
We can also provide users with this forked pgvector repo, which is customized for Cloudberry: https://github.com/cloudberry-contrib/pgvector, but not maintained by the Cloudberry PPMC.